src/test/java/seedu/contax/testutil/DateInputUtil.java

```diff
@@ -0,0 +1,31 @@
+package seedu.contax.testutil;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.contax.commons.util.DateUtil;
+
+public class DateInputUtil {
```

Comment:
Missing javadoc

Reply:
Thanks for catching it, fixed in commit f9b8464,